modules/ROOT/pages/anypoint-mq/3.x/anypoint-mq-listener.adoc

```diff
@@ -0,0 +1,347 @@
+| *(3)* | Specifies how long to wait before resuming new message polling after the circuit breaker trips because the `errorsThreshold` is reached.
+| *(4)* | Defines the operation to throw the error expected by the `onErrorTypes` parameters.
+|===
+
+The circuit breaker ignores all errors that aren't listed in the `onErrorTypes` parameter. In this example, the circuit breaker ignores errors such as `HTTP:BAD_REQUEST`.
+
+
+====  Share a Circuit from Different Queues
+
+In many cases, there's a single common service between the processing of messages from different queues. This example configures the `circuitName` parameter to bind both subscribers to a single circuit:
+
+[source,xml,linenums]
+----
+
+<anypoint-mq:config name="Anypoint_MQ_Config">
+    <anypoint-mq:connection url="${providerUrl}"
+       clientId="${clientId}"
+       clientSecret="${clientSecret}"/>
+</anypoint-mq:config>
+
+<anypoint-mq:circuit-breaker
+    name="InvoiceProcess"                    <!-- (1) -->
+    onErrorTypes="FTP:RETRY_EXHAUSTED, HTTP:SERVICE_UNAVAILABLE" <!-- (2) -->
+    errorsThreshold="10"
+    tripTimeout="5"
+    tripTimeoutUnit="MINUTES"/>
+
+<flow name="subscribe">
+    <anypoint-mq:subscriber destination="${reservationsQueue}"
+       config-ref="Anypoint_MQ_Config"
+        circuitBreaker="GlobalCircuit"/>     <!-- (3) -->
+    <flow-ref name="invoiceProcess">
+</flow>
+
+<flow name="otherSubscribe">
+    <anypoint-mq:subscriber
+      destination="${paymentsQueue}"
+        config-ref="Anypoint_MQ_Config"
+        circuitBreaker="GlobalCircuit"/>    <!-- (3) -->
+    <flow-ref name="invoiceProcess">
+</flow>
+
+<sub-flow name="invoiceProcess">
+  <ftp:write path="${auditFolder}"          <!-- (4) -->
+	   config-ref="ftp-config"/>
+  <http:request config-ref="requestConfig"  <!-- (5) -->
+	   path="/external"/>
+</sub-flow>
+
+----
+
+[%noheader,cols="8,92,frame="none"]
+|===
+| *(1)* | Sets the `name` parameter to share a common circuit breaker on multiple queues.
+| *(2)* | Identifies 2 errors that can affect the processing of messages from the subscriber and passes each as a CSV list.
+| *(3)* | For both subscribers, references the global circuit breaker configuration.
+| *(4)* | This component might throw an `FTP:RETRY_EXHAUSTED` error along with many others. Only the `FTP:RETRY_EXHAUSTED` error is taken into account by the circuit breaker.
+| *(5)* | The HTTP connector might throw an `HTTP:SERVICE_UNAVAILABLE` preventing the message to be processed.
+|===
+
+For this scenario, both subscribers stop polling for messages as soon as the error count reaches the `errorsThreshold="10"` value, counting both `FTP:RETRY_EXHAUSTED` and `HTTP:SERVICE_UNAVAILABLE` errors. When the `tripTimeout` elapses, one of the subscribers polls for a message and uses it to test the circuit, enabling the polling for both subscribers if the processing of that message succeeds.
```

Comment:
In this scenario, both

Reply:
...to test the circuit. If the message succeeds, polling for both subscribers is enabled.   ("resumed"?)